qt_gui/src/qt_gui/help_provider.py

```diff
@@ -47,7 +46,18 @@ def plugin_help_request(self, plugin_descriptor):
         package_name = plugin_descriptor.attributes()['package_name']
         package_path = get_package_path(package_name)
         try:
-            manifest = parse_manifest_file(package_path, MANIFEST_FILE)
-        except (InvalidManifest, IOError):
+            package = parse_package(package_path)
+        except (InvalidPackage, IOError):
+            return
+
+        if not package.package_format or package.package_format < 2:
+            return
```

Comment:
Why is this check necessary? The following code seems to be fine for package format 1.

Reply:
Removed check